server/src/main/java/org/elasticsearch/common/util/CancellableThreads.java

```diff
@@ -167,11 +167,11 @@ public synchronized void cancel(String reason) {
     }
 
 
-    public interface Interruptable extends IOInterruptable {
+    public interface Interruptible extends IOInterruptible {
         void run() throws InterruptedException;
     }
 
-    public interface IOInterruptable {
+    public interface IOInterruptible {
```

Comment:
This is a `public` interface, it's also misspelled. I can:
* drop the change
* split the change to its own pr
* include a shim